### DISCUSSION TITLE
2023/07/27/introduce-snippet/

### DISCUSSION THREAD

**Discussion by giscus** [cloudmatelabs/tech.cloudmt.co.kr#168] [Blog Comments]
# 2023/07/27/introduce-snippet/

Snippet은 ‘코드 조각’ 이라는 뜻입니다. 클라우드메이트에서 만든 Snippet 서비스를 소개하려고 합니다. 소개하려는 것: https://snippet.cloudmt.co.kr/ 혼자서

https://tech.cloudmt.co.kr/2023/07/27/introduce-snippet/

**Comment by becta012** [comment]
음, 필요한걸 찾다 화가 난다는데 왠지 공감 되네요...^^;;
그래도 요즘은 chatGPT나 Bing이 많은 도움이 되더라고요.
비록 잘못 알려줘서 삽질(?) 할때가 간간히 있긴하지만...

**Reply by juunini** [reply]
@KimALam
클라우드 엔지니어링 관련으로 물어보면
틀린걸 얼려주거나 이상한 답을 해서 힘들더라구요 ㅠㅠ

**Comment by r4you96** [comment]
너무 좋은 서비스인거같은데 더 홍보가 되어서 활성화되면 좋겠네요
